app/src/full/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -134,6 +141,29 @@ class MessagingService : FirebaseMessagingService() {
         }
     }
 
+    private fun speakNotification(title: String?) {
+        var textToSpeech: TextToSpeech? = null
+        var tts = title
+        if (tts.isNullOrEmpty())
+            tts = getString(R.string.tts_no_title)
+        textToSpeech = TextToSpeech(applicationContext
```

Comment:
We can probably make this a val here and remove the var above.
```suggestion
        val textToSpeech = TextToSpeech(applicationContext
```

Reply:
When I test this change locally the variable below where we call `textToSpeech?` has an error, I assume because its in the same method here?